functions/go/apply-replacements/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"os"
+
+	sdk "github.com/GoogleContainerTools/kpt-functions-catalog/thirdparty/kyaml/fnsdk"
```

Comment:
this fnsdk is experimental library, we eventually should use the one from kpt-functions-sdk. Can you add a TODO here and to the  kpt-functions-catalog/thirdparty/kyaml/fnsdk as well so we know this function need to be updated when deprecating kpt-functions-catalog/thirdparty/kyaml/fnsdk?

Reply:
Now using kpt-functions-sdk!